source/ConfigSetup.h

```diff
@@ -24,6 +24,7 @@ EMP_BUILD_CONFIG(SymConfigBase,
     VALUE(SYM_HORIZ_TRANS_RES, double, 100, "How many resources required for symbiont non-lytic horizontal transmission"),
     VALUE(SYM_VERT_TRANS_RES, double, 0, "How many resources required for symbiont vertical transmission"),
     VALUE(GRID, bool, 0, "Do offspring get placed immediately next to parents on grid, same for symbiont spreading"),
+    VALUE(RANDOM_INJECTION, bool, 1, "Should organisms be randomly injected into the world or should they be consistently spaced out?"),
```

Comment:
The comment should mention that 1 is the first choice and 0 is the second choice